test/JDBC/expected/BABEL-937.out

```diff
@@ -0,0 +1,210 @@
+-- Not Append
+--NOT NULL, LAX, KP  
+SELECT JSON_MODIFY('{"name":"John","skills":["C#","SQL"]}','$.name','Mike');
+GO
+~~START~~
+nvarchar
+{"name": "Mike", "skills": ["C#", "SQL"]}
+~~END~~
+
+
+--NOT NULL, STRICT, KP
+SELECT JSON_MODIFY('{"name":"John","skills":["C#","SQL"]}','strict $.name','Mike');
```

Comment:
Need to test upper/mixed case, as well as extra spaces, leading/trailing spaces etc. Even if those aren't supported we should know what error they're returning.

Reply:
Have tested the upper/mixed case and also leading/trailing spaces and have handled them in the implementation. Also added the test cases in the BABLE-937.sql file.